tsconfig.json

```diff
@@ -7,13 +7,16 @@
 		"module": "commonjs",
 		"moduleResolution": "node",
 		"outDir": "out",
-		"sourceMap": true
+		"sourceMap": true,
+		"resolveJsonModule": true,
```

Comment:
**"sourceMap"** - not needed, I tested some stuff around and then I forget to remove that

**"resolveJsonModule"** - is needed for possibility of UI testing properties from 'package.json' file directly according to latest version ('package.json' file vs displayed extension info). But this property also changing hierarchy of 'out/' folder. I don't know completely why, but my humble tip is because of raw project hierarchy where 'package.json' has different location then extension.ts and others... 

**"esModuleInterop": true** - was needed as consequence of resolveJsonModule compile errors but I found solution in different syntax 

from
```
import pjson from ....
```
to
```
import * as pjson from .....
```